spyder_kernels/ipdb/kernel.py

```diff
@@ -184,8 +184,11 @@ def get_usage(self):
 
     def get_kernel_help_on(self, info, level=0, none_on_fail=False):
         """Help for Pdb commands."""
-        cmd = info['code'].strip()
-        return self.debugger.do_help(cmd)
+        if none_on_fail:
+            return None
+        else:
+            cmd = info['code'].strip()
+            return self.debugger.do_help(cmd)
```

Comment:
This avoid the error I reported and it's nice. But what I really want is to show the same help as it's shown by qtconsole when it's using ipykernel.

And according to this

https://github.com/ipython/ipykernel/blob/e32d7f9360e345e87bc4f10dbb66831421b0900b/ipykernel/ipkernel.py#L401

it shouldn't be too hard to implement, now that we have an `ipyshell` object in the IPdb kernel.